app/script_variant.py

```diff
@@ -0,0 +1,80 @@
+import jautils
+
+from unidecode import unidecode
+from google.appengine.api import memcache
+
+import re
+import logging
+
+def read_dictionary(file_name):
+    dictionary = {}
+    for line in open(file_name, 'r'):
+        kanji, hiragana = line[:-1].split('\t')
+        dictionary[kanji.decode('utf-8')] = hiragana.decode('utf-8')
+    return dictionary
+
+JAPANESE_NAME_DICTIONARY = read_dictionary('japanese_name_dict.txt')
+JAPANESE_LOCATION_DICTIONARY = read_dictionary('jp_location_dict.txt')
+
+def romanize_word(word):
```

Comment:
Can you move this function just above romanize_text(), so the structure of the functions is clearer.